packages/psss/src/containers/Tables/CountryReportTable.js

```diff
@@ -116,82 +160,82 @@ export const CountryReportTable = React.memo(
       setTableStatus(TABLE_STATUSES.STATIC);
     }, [setTableStatus]);
 
-    if (Object.keys(fields).length === 0) {
-      return null;
-    }
-
     return (
       <LoadingContainer
         isLoading={isFetching || tableStatus === TABLE_STATUSES.SAVING}
         heading={isFetching ? 'Loading data' : 'Saving Data'}
       >
-        <FlexSpaceBetween pb={2}>
-          {tableStatus === TABLE_STATUSES.EDITABLE ? (
-            <FormRow>
-              <ReportedSites variant="h6">Reported Sites:</ReportedSites>
-              <StyledTextField
-                value={sitesReportedValue}
-                onChange={event => setSitesReportedValue(event.target.value)}
-                name="sites-reported"
-              />
-              <ReportedSites variant="h5"> / Total Sites: </ReportedSites>
-              <StyledTextField
-                value={totalSitesValue}
-                onChange={event => setTotalSitesValue(event.target.value)}
-                name="total-sites"
-              />
-            </FormRow>
-          ) : (
-            <Typography variant="h5">
-              {sitesReportedValue}/{totalSites} Sites Reported
-            </Typography>
-          )}
-          <GreyOutlinedButton
-            onClick={handleEdit}
-            disabled={tableStatus === TABLE_STATUSES.EDITABLE}
-          >
-            Edit
-          </GreyOutlinedButton>
-        </FlexSpaceBetween>
-        {/*ToDo: implement with sentinel sites feature*/}
-        {/*@see https://app.zenhub.com/workspaces/sprint-board-5eea9d3de8519e0019186490/issues/beyondessential/tupaia-backlog/1640*/}
-        {/*{tableStatus === TABLE_STATUSES.EDITABLE && (*/}
-        {/*  <Alert severity="error" variant="standard">*/}
-        {/*    Updating country level data manually: all individual sentinel site data will be ignored*/}
-        {/*  </Alert>*/}
-        {/*)}*/}
-        {/*<GreyAlert severity="info" icon={<InfoIcon fontSize="inherit" />}>*/}
-        {/*Country level data has been manually edited, sentinel data will not be used.*/}
-        {/*</GreyAlert>*/}
-        <GreyHeader>
-          <span>SYNDROMES</span>
-          <span>TOTAL CASES</span>
-        </GreyHeader>
-        <EditableTable Header={false} Body={VerifiableBody} />
-        {tableStatus === TABLE_STATUSES.EDITABLE && (
-          <FlexEnd pt={3} mt={3} borderTop={1} borderColor="grey.400">
-            {/*<MuiLink underline="always">Reset and use Sentinel data</MuiLink>*/}
-            <div>
-              <Button variant="outlined" onClick={handleCancel}>
-                Cancel
-              </Button>
-              <Button onClick={handleSubmit}>Save</Button>
-            </div>
-          </FlexEnd>
-        )}
+        <FormProvider {...methods}>
+          <form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
make this an actual html form and use ReactHookForm to track the data for the form submission.